dbsim/db_server.cpp

```diff
@@ -50,31 +48,18 @@ logger_fn(wsrep::log::level l, const char* pfx, const char* msg)
              date.tm_year + 1900, date.tm_mon + 1, date.tm_mday,
              date.tm_hour, date.tm_min, date.tm_sec, (int)time.tv_nsec/1000000);
 
-#define LOG_STR date_str << pfx << wsrep::log::to_c_string(l) << msg
-    if (l >= wsrep::log::error && reporter)
-    {
-        std::ostringstream os;
-        os << LOG_STR;
-        std::cerr << os.str() << std::endl;
-        auto const tstamp(double(time.tv_sec) + double(time.tv_nsec)*1.0e-9);
-        reporter->report_log_msg(wsrep::reporter::error, os.str(), tstamp);
-    }
-    else
-    {
-        std::cerr << LOG_STR << std::endl;
-    }
-#undef LOG_STR
+    std::cerr << date_str << pfx << wsrep::log::to_c_string(l) << msg << std::endl;
```

Comment:
Please add spaces between date_str and pfx, log level string and msg to make log more.readaböe. Now the lines look like:
```
2021-09-12 15:29:57.105L:infoConnecting with bootstrap option: 0
```